src/Pages/Withdrawals.razor

```diff
@@ -772,6 +796,50 @@
         }
     }
 
+    private async Task ShowBumpfeeModal(WalletWithdrawalRequest walletWithdrawalRequest)
```

Comment:
There's a edge case where you fail the child tx and the modal cannot be bumped for nor the parent or child

Reply:
I will check and resolve